src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -250,6 +254,7 @@ export class ExtensionEditor extends BaseEditor {
 		this.navbar.onChange(this.onNavbarChange.bind(this, extension), this, this.transientDisposables);
 		this.navbar.push(NavbarSection.Readme, localize('details', "Details"));
 		this.navbar.push(NavbarSection.Contributions, localize('contributions', "Contributions"));
+		this.navbar.push(NavbarSection.Changelog, localize('changelog', "Changelog"));
```

Comment:
@joaomoreno 
I'm a bit sad for that method, but that's the life!
Fixed in b98c3f7